source/get-started/download-and-install.txt

```diff
@@ -0,0 +1,79 @@
+.. _ruby-quick-start-download-and-install:
+
+====================
+Download and Install
+====================
+
+.. facet::
+   :name: genre
+   :values: tutorial
+
+.. meta::
+   :keywords: installation, setup, code example
+
+.. procedure::
+   :style: connected
+
+   .. step:: Install dependencies
+
+      Before you begin developing, ensure you have the following dependencies
+      installed in your development environment:
+      
+      - `Ruby <https://www.ruby-lang.org/en/downloads/>`__ version 2.7
+        or later
+      - `bson <https://rubygems.org/gems/bson>`__ version 4.14.1 or later, up to
+        version 6.0
+
+      .. note::
+
+         {+language+} is pre-installed on macOS and some Linux distributions,
+         but the system version might be outdated. You can use a package 
+         manager, such as `Homebrew <https://formulae.brew.sh/formula/ruby>`__,
+         to install the latest {+language+} version.
+
+   .. step:: Install the {+driver-short+}
+
+      From your shell, run the following command to install
+      the {+driver-short+}:
+
+      .. code-block:: bash
+         
+         gem install mongo
```

Comment:
Installing gems using `gem install` may fail if a system wide Ruby installation is used. It is also considered as not the best practice in general nowadays. 

It might be better to use [bundler](https://bundler.io/guides/getting_started.html) for managing the getting started project. Any modern distribution of Ruby comes with Bundler preinstalled by default.

If the project is going to be a single file, then [inlined bundler](https://bundler.io/guides/bundler_in_a_single_file_ruby_script.html) is ideal:

```ruby
require 'bundler/inline'

gemfile do
  source 'https://rubygems.org'
  gem 'mongo'
end
```

Reply:
It's just a single file, so I will add inlined bundler instructions